node/node.go

```diff
@@ -89,7 +89,9 @@ import (
 )
 
 var (
-	genesisHashKey  = []byte("genesisID")
+	genesisHashKey = []byte("genesisID")
+	sessionKey     = []byte("session")
```

Comment:
```suggestion
	gracefulShutdownKey     = []byte("gracefulShutdown")
```

I think this naming makes things a bit more clear but w/e either way